lib/src/main/java/com/cloudbees/groovy/cps/Next.java

```diff
@@ -141,7 +141,7 @@ public static Next terminate0(Outcome v) {
     /**
      * As a {@link Continuation}, just ignore the argument.
      */
-    public Next receive(Object _) {
+    public Next receive(Object unused) {
```

Comment:
These changes are also not strictly necessary right now, but will be needed to be able to compile against Java 11 in the future, and this seems like as good a time as any to make the change.

Reply:
Is there a reason you are not just going to 11 now?

Reply:
If we generate Java 11 bytecode for `groovy-cps`, then `workflow-cps` would require Java 11, and my understanding is that not many (if any?) plugins require Java 11 right now, and I am not really interested in experimenting with a widely-installed plugin like `workflow-cps`.

Reply:
Full support for Java 11 in the plugin toolchain will need to wait a few more months until 2.361 LTS and future are more widely adopted.

Reply:
> `workflow-cps` would require Java 11

Well, yes, a 2.361.x baseline.

> not many (if any?) plugins require Java 11 right now

Actually there are a fair number already on 2.361.x (or even subsequent weeklies), mainly because of core UI changes.